framework/include/media/FileOutputDataSource.h

```diff
@@ -106,9 +106,15 @@ class FileOutputDataSource : public OutputDataSource
 	 * @since TizenRT v2.0 PRE
 	 */
 	bool close() override;
-
-protected:
-	ssize_t onStreamBufferReadable(bool isFlush) override;
+	/**
+	 * @brief Write the file
+	 * @details @b #include <media/FileOutputDataSource.h>
+	 * param[in] buf poiter to a buffer
+	 * param[in] size Number of size to write
+	 * @return True.
```

Comment:
It seems that "True" needs to be changed as something like "Data size written to buf".

Reply:
Fixed it